files/scripts/swss.sh

```diff
@@ -306,6 +306,15 @@ function check_peer_gbsyncd()
     fi
 }
 
+function check_macsec()
+{
+    MACSEC_STATE=`show feature status | grep macsec | awk '{print $2}'`
+
+    if [[ ${MACSEC_STATE} == 'enabled' ]]; then
+        DEPENDENT="${DEPENDENT} macsec"
```

Comment:
does not correct it should be like this https://github.com/sonic-net/sonic-buildimage/commit/23dcfdf9b672a1f637f618741ca26c31490fe623

Otherwise i think multi-asic might now work

Reply:
Make sense, Thanks for your suggestion.
@judyjoseph Could you help to check the latest commit agian.

Reply:
Yes it is fine now, thanks to @abdosi for pointing it out.
Btw @Pterosaur may be this branch needs a rebase the script don't have the bgp changes.

Reply:
Yes, Done.